src/Api/Auth/Controllers/WebAuthnController.cs

```diff
@@ -14,67 +17,101 @@ namespace Bit.Api.Auth.Controllers;
 public class WebAuthnController : Controller
 {
     private readonly IUserService _userService;
+    private readonly IWebAuthnCredentialRepository _credentialRepository;
+    private readonly IDataProtectorTokenFactory<WebAuthnCredentialCreateOptionsTokenable> _createOptionsDataProtector;
 
     public WebAuthnController(
-        IUserService userService)
+        IUserService userService,
+        IWebAuthnCredentialRepository credentialRepository,
+        IDataProtectorTokenFactory<WebAuthnCredentialCreateOptionsTokenable> createOptionsDataProtector)
     {
         _userService = userService;
+        _credentialRepository = credentialRepository;
+        _createOptionsDataProtector = createOptionsDataProtector;
     }
 
     [HttpGet("")]
-    // TODO: Create proper models for this call
-    public async Task<ListResponseModel<TwoFactorWebAuthnResponseModel>> Get()
+    public async Task<ListResponseModel<WebAuthnCredentialResponseModel>> Get()
     {
         var user = await _userService.GetUserByPrincipalAsync(User);
         if (user == null)
         {
             throw new UnauthorizedAccessException();
         }
-        return new ListResponseModel<TwoFactorWebAuthnResponseModel>(new List<TwoFactorWebAuthnResponseModel> { });
+
+        var credentials = await _credentialRepository.GetManyByUserIdAsync(user.Id);
+
+        return new ListResponseModel<WebAuthnCredentialResponseModel>(credentials.Select(c => new WebAuthnCredentialResponseModel(c)));
     }
 
     [HttpPost("options")]
     [ApiExplorerSettings(IgnoreApi = true)] // Disable Swagger due to CredentialCreateOptions not converting properly
```

Comment:
I think we can remove this now

Reply:
`CredentialCreateOptions` is still part of the response model as a property on that class. Swagger seems to have no problem with it though, so I'll remove!